pkg/client/e2e.go

```diff
@@ -57,35 +56,35 @@ func (*SonobuoyClient) GetTests(reader io.Reader, show string) ([]reporters.JUni
 	}
 
 	if !found {
-		return nil, fmt.Errorf("failed to find results file %q in archive", e2eJunitPath)
+		return nil, fmt.Errorf("failed to find results file %q in archive", e2eJUnitPath)
 	}
 
-	out := make([]reporters.JUnitTestCase, 0)
+	out := make([]results.JUnitTestCase, 0)
 	if show == "passed" || show == "all" {
-		out = append(out, results.Filter(results.Passed, junitResults)...)
+		out = append(out, results.JUnitFilter(results.JUnitPassed, junitResults)...)
```

Comment:
Although not strictly requiring this prefix now that there isnt a `junit` and a `e2e` filter/passed/etc, it makes sense to me to leave a prefix since this _is_ type specific.

As we add other formats we will probably need similar named things. At that point we may also want to consider some interface to do this instead though.